iojs.org/aliases.json

```diff
@@ -31,7 +44,8 @@
     "thechargingvolcano@gmail.com",
     "targos@protonmail.com",
     "joyeec9h3@gmail.com",
-    "matteo.collina@gmail.com"
+    "matteo.collina@gmail.com",
+    "josh.gavant@gmail.com"
```

Comment:
Do we need to keep/maintain the duplicate list for ctc? Can it be aliased to tsc?

Reply:
heh. I forgot until after that we could do that :-) ... yeah, I'll make ctc and alias for tsc